.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug Report
+about: Help us diagnose and fix Upbound bugs
+labels: bug, needs-triage
+---
+<!--
+Thank you for helping to improve Upbound! We use issues for bug reports and
+feature requests. Please be sure to search for open issues before raising a new
+one.
+-->
+
+### What happened?
+<!--
+Please let us know what behaviour you expected and how Upbound diverged from
+that behaviour.
+-->
+
```

Comment:
I considered adding a checklist for the issue templates, which would include:

* Adding a size estimate label to your issue.
* Updating the relevant epic to link to the new issue.

I feel like we could automate a lot of this though. Given that GitHub Actions are mostly written in TypeScript it might be worth us putting a day aside for those of us with TS and GitHub Actions experience to hack on this. I suspect we'd be able to make everyone's lives a lot easier with a day or so spent on building automation.

Reply:
Specifically https://github.com/mathiasjakobsen/github-estimates-action and https://github.com/kiali/kiali-epic-action go a long way toward aggregating issue estimates and keeping epics linked to sub-tasks, but unfortunately neither appear to support an org wide model in which epics might be in a different repo and project boards might be at the org level.